3_run_rsync/vars/defaults.yml

```diff
@@ -16,5 +16,5 @@ stunnel_key_filepath: "./files/tls.key"
 rsyncd_port:  22
 
 # Rsync default options
-rsync_opts: "-aPvvH --delete"
+rsync_opts: "-aPvvHh --delete"
```

Comment:
@pranavgaikwad will '-h' have any implications to the pvc resizing work.
Wanted to check with you since I think -h is for 'human readable'.
Not sure if you are relying on any output from rsync with the unit conversion work

Reply:
@pranavgaikwad @jwmatthews Let me know, as far as the flag is concerned it will just display rsync stat output on human readable but not sure if we are relying on rsync for something or if we just depend on "df" and "PVC" output.

Reply:
@fbladilo I am not using the `rsync` output for pvc size estimation. this should be good